cmake/TaichiCore.cmake

```diff
@@ -223,10 +223,16 @@ if (TI_WITH_VULKAN)
       find_library(SPIRV_OPT NAMES "SPIRV-Tools-opt" PATHS "${Vulkan_INCLUDE_DIR}/../Lib" REQUIRED)
       find_library(SPIRV_LINK NAMES "SPIRV-Tools-link" PATHS "${Vulkan_INCLUDE_DIR}/../Lib" REQUIRED)
     else ()
-      set(SPIRV_OPT SPIRV-Tools-opt)
-      set(SPIRV_TOOLS SPIRV-Tools)
-      set(SPIRV_LINK SPIRV-Tools-link)
+      get_filename_component(VULKAN_LIBRARY_PATH ${Vulkan_LIBRARY} DIRECTORY)
+      find_library(SPIRV_TOOLS NAMES "SPIRV-Tools"  PATHS ${VULKAN_LIBRARY_PATH} REQUIRED)
+      find_library(SPIRV_OPT NAMES "SPIRV-Tools-opt" PATHS ${VULKAN_LIBRARY_PATH} REQUIRED)
+      find_library(SPIRV_LINK NAMES "SPIRV-Tools-link"  PATHS ${VULKAN_LIBRARY_PATH} REQUIRED)
```

Comment:
Is this an variable from the environment? Or is it a manually set flag

Reply:
See L226. The directory path is extracted from ${Vulkan_LIBRARY}.

Reply:
Okay sorry missed that